pkg/cmd/cli/cmd/loginoptions.go

```diff
@@ -159,6 +159,13 @@ func (o *LoginOptions) getClientConfig() (*restclient.Config, error) {
 				fmt.Fprintln(o.Out)
 			}
 
+		// path "/" we tried to ping is forbidden, but the server is still reachable
+		case clientcmd.IsForbidden(result.Error()):
```

Comment:
@smarterclayton Updated to ping the root of OpenShift API set in `config.APIPath` (`/oapi` at the moment). @liggitt Removed the `IsForbidden` check. PTAL.